doc/Quick_Start/README.md

```diff
@@ -285,15 +289,17 @@ Just connect them to the IPEX connector of the RAK4631 or RAK1910.
 
 ### Battery
 
-Please use the battery which RAK suggest, and connect the battery with RAK5005-O as follow:
+Please use the battery which RAK suggest, and connect the battery with RAK5005-O as follow:     
 
 <center><img src="../../assets/repo/battery-connect.png" alt="battery-connect" width="50%"/></center>
 
 # When connecting the battery make sure the polarity is correct. Not all connectors are wired the same.
 # Ground pole should be close to the PCB boarder.
 # Plus pole should be close to USB connector.
```

Comment:
You can add a header "Important Reminders:", so that you can justify the usage of BOLD characters.

Reply:
Yes, it was too large. Here a better solution with your proposal:
![image](https://user-images.githubusercontent.com/512690/97516688-a230da00-19ce-11eb-8307-113ac6f7fc60.png)